EmbedScratch.php

```diff
@@ -31,7 +31,7 @@ public static function parserEmbedScratch (&$parser) {
 	    return true;
 	}
 	
-	function renderEmbedScratch ($input, $argv, $parser) {
+	static function renderEmbedScratch ($input, $argv, $parser) {
```

Comment:
I think if we really want to do this right, this should also be public.

Reply:
Well sure but that seems unnecessary.

Reply:
If we want the function to be accessible outside of the class, we should be explicit about it.